src/ComScoreStreamingAnalytics.ts

```diff
@@ -195,15 +238,15 @@ export class ComScoreStreamingAnalytics {
       ns_st_sn: this.metadata.episodeSeasonNumber,
       ns_st_en: this.metadata.episodeNumber,
       ns_st_ge: this.metadata.contentGenre,
-      ns_st_ddt: this.metadata.digitalAirdate,
-      ns_st_tdt: this.metadata.tvAirdate,
-      ns_st_st: this.metadata.stationTitle,
+      ns_st_ddt: this.metadata.digitalAirdate ? this.metadata.digitalAirdate : '*null',
+      ns_st_tdt: this.metadata.tvAirdate ? this.metadata.tvAirdate : '*null',
+      ns_st_st: this.metadata.stationTitle ? this.metadata.stationTitle : '*null',
       c3: this.metadata.uniqueContentId,
       c4: this.metadata.uniqueContentId,
       c6: this.metadata.uniqueContentId,
       ns_st_ft: this.metadata.feedType,
-      ns_st_ce: this.metadata.completeEpisode ? '1' : null,
-      ns_st_ia: this.metadata.advertisementLoad ? '1' : null,
+      ns_st_ce: this.metadata.completeEpisode ? this.metadata.completeEpisode : '*null',
```

Comment:
What is this `*null` value and why not using `null` like before?

If `*null` should be used please extract this to a `const` variable and use the variable instead